api/app/reviews/api.py

```diff
@@ -238,4 +240,57 @@ def validate_scores(self, scores):
                 return self.get_error_message('value')
     
     def get_error_message(self, key):
-        return ({'message': {key: 'Missing required parameter in the JSON body or the post body or the query string'}}, 400)
\ No newline at end of file
+        return ({'message': {key: 'Missing required parameter in the JSON body or the post body or the query string'}}, 400)
+
+class ReviewAssignmentAPI(PostReviewAssignmentMixin, restful.Resource):
+    
+    @auth_required
+    def post(self):
+        args = self.post_req_parser.parse_args()
+        user_id = g.current_user['id']
+        event_id = args['event_id']
+        reviewer_user_email = args['reviewer_user_email']
+        num_reviews = args['num_reviews']
+
+        current_user_roles = self.get_roles(user_id, event_id)
+        if 'event_admin' not in current_user_roles and 'system_admin' not in current_user_roles:

```

Comment:
Is `event_admin` and `system_admin` the correct the role names?

Reply:
"System admin" isn't a role, it's when the is_admin flag on the user object is True

Reply:
And the event admin role is just "admin" (since it's already in the EventRole table, the event_ prefix is redundant)

Reply:
Ah I see now. Thanks. Done.